google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPoolOptions.java

```diff
@@ -271,6 +285,18 @@ long getRandomizePositionQPSThreshold() {
     return randomizePositionQPSThreshold;
   }
 
+  boolean getUseMultiplexedSession() {
```

Comment:
I assume that we are making these method package-private because we don't want to expose these methods publicly yet, but that they will be made public at a later time.

Reply:
Yes, you are right. Will make it public at a later point.